packages/backend/app.js

```diff
@@ -9,6 +9,6 @@ app.use(cors());
 
 app.use(express.static("webui/dist"));
 app.use("/api", apiRouter);
-app.use("/nixosConfigurations", express.static("./webui/nixosConfigurations"));
+app.use("/nixosConfigurations", express.static("../../webui/nixosConfigurations"));
```

Comment:
Notice that this has to be tried after compiling the frontend. When you talk about the relative path, notice that once you have compiled the binary, then the current path might be the flake root path, for which this change might actually break things.

Does it? I don't know, it depends on how nodejs interprets relative paths in this case. Another solution, if necessary, is to use the current working directory path here. Let me know if that is necessary, it will need some additional parameters to be brought into scope.